model/src/wav_history_mod.F90

```diff
@@ -446,6 +452,15 @@ subroutine write_history ( timen )
 
     call pio_closefile(pioid)
 
+    ! create indicator log file after NetCDF file is written
```

Comment:
What if a user does not want these files generated?

Reply:
These files are required to meet operational requirements see https://github.com/NOAA-EMC/WW3/issues/921 for additional details.  A feature could be added in the future to make this optional if needed.

Reply:
Optional means UFS could have enabled it, if required for opn.